src/dstore/pmix_esh.c

```diff
@@ -33,13 +33,26 @@
 #include "pmix_dstore.h"
 #include "pmix_esh.h"
 
+typedef struct nspace_tbl_entry_s {
```

Comment:
@karasevb 
I guess this is more users table and not namespace. Lets rename it for consistency.